CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v4.7.1 - 2023-11-01
+
+- Adding explicit type for Optional OneToOneField [#113](https://github.com/octoenergy/xocto/pull/113)
```

Comment:
In semver, this is a 4.8.0 change, not 4.7.1

Reply:
✅